pkg/ingester/user_state_test.go

```diff
@@ -0,0 +1,42 @@
+package ingester
+
+import (
+	"testing"
+
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/pkg/labels"
+
+	"github.com/stretchr/testify/require"
+	"github.com/weaveworks/common/user"
+	"golang.org/x/net/context"
+)
+
+func TestForSeriesMatching(t *testing.T) {
```

Comment:
Can you name it `TestForSeriesMatchingBatching` as batching only is being tested?

Reply:
Sure!